### PR TITLE
fix: adding check if the window exists in the browser

### PR DESCRIPTION
**Pull request by ramonxm**
## Description
This will show the hidden texts of the animation.

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

**Comment by jonataspinto** [comment]
image


Can you include @ramonxm ?

**Comment by jonataspinto** [comment]
My preference is to rename this file.

**Review by jonataspinto — APPROVED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -22,13 +22,15 @@ import {
...
 const Home = () => {
   useEffect(() => {
-    Aos.init({
-      delay: 50,
-      offset: 200,
-      once: false,
-      mirror: true,
-      duration: 1000,
-    });
+    if (typeof window !== 'undefined') {
```
Comment (jonataspinto) [inline src/pages/index.tsx:25]: I did a test simulating `user-agent` as `googlebot` and the problem persists :/

Let's try this here in this `useEffect` ? https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent#examples

```js
if(useAgent !== {GOOGLEBOT}) // init Aos
```

Comment (ramonxm) [reply src/pages/index.tsx:25]: I will try here

**Review by jonataspinto — APPROVED** [review]

**Pull request merged**